### ISSUE TITLE
Question about the metrics

### ISSUE THREAD

**Issue by PAIplus** [A4Bio/RDesign#3]
https://github.com/A4Bio/RDesign/blob/b61988824d9e69cde316e0c593c446711f0ce36f/methods/rdesign.py#L60
We found that "np.median" is used in the metrics. Is it the result reported in the paper? Or we should change it into np.mean ?

**Comment by chengtan9907** [comment]
Yes. We use `np.median` for each experiment, and repeat three times with different randoms. Then, we present the average and standard deviation of these three iterations as shown in the paper.
